generators/typescript/sdk/generator/src/SdkGenerator.ts

```diff
@@ -409,7 +409,8 @@ export class SdkGenerator {
                 run: ({ sourceFile, importsManager }) => {
                     const context = this.generateSdkContext({ sourceFile, importsManager });
                     context.type.getGeneratedType(typeDeclaration.name).writeToFile(context);
-                }
+                },
+                addExportTypeModifier: true
```

Comment:
can we add this to request wrappers instead of types declarations?

Reply:
https://github.com/hookdeck/hookdeck-typescript-sdk/blob/ecda54dde80df5286f460d5f15833ae17f299d99/src/api/resources/bookmark/client/requests/index.ts#L1-L4 is an example of the first problem to fix

Reply:
all request wrappers are interfaces so we know that this is safe